Detectors/Base/include/DetectorsBase/Detector.h

```diff
@@ -95,18 +109,18 @@ class Detector : public FairDetector
 
 
     /// Mapping of the ALICE internal material number to the one
-    /// automatically assigned by geant.
-    /// This is required for easily being able to copy the geometry setup
+    /// automatically assigned by geant/TGeo.
+    /// This is required to easily being able to copy the geometry setup
     /// used in AliRoot
-    std::unique_ptr<std::vector<int>> mMapMaterial; //!< material mapping
+    std::map<int, int> mMapMaterial; //! material mapping
```

Comment:
For doxygen I think the comment should be //!< to make clear the comment belongs to the line before not after. Same below.